packages/firebase_remote_config/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.2]
+
+* Fix iOS casting issue.
```

Comment:
This is a bit vague, we might want to describe the APIs or behavior that was previously broken and now works

Reply:
Done